packages/vite-plugin-cloudflare/src/cloudflare-environment.ts

```diff
@@ -167,7 +172,10 @@ export function createCloudflareEnvironmentOptions({
 			copyPublicDir: false,
 			ssr: true,
 			rollupOptions: {
-				input: workerConfig.main,
+				input: {
+					// Changes to this file name must be reflected when populating `main` in the `generateBundle` hook in `index.ts`
+					index: VIRTUAL_WORKER_ENTRY,
```

Comment:
If I understand this correctly, this key must match `workerConfig.main`, right? Should we make it a shared constant too? e.g.

```ts
input: {
 [MAIN_ENTRY]: VIRTUAL_WORKER_ENTRY,
}
```